extensions/amp-video/0.1/validator-amp-video.protoascii

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # <amp-video>
+  html_format: AMP
+  html_format: AMP4ADS
+  tag_name: "AMP-VIDEO"
+  disallowed_ancestor: "AMP-SIDEBAR"
+  also_requires_tag_warning: "amp-video extension .js script"
```

Comment:
@cramforce This rule is the one that creates the warning if users don't include the <script> tag.